### ISSUE TITLE
Run on ECS Fargate as a container

### ISSUE THREAD

**Issue by aecceyhan** [knadh/listmonk#1131]
Hello All,

I am trying to run the platform on my ECS cluster. So I basically https://hub.docker.com/r/listmonk/listmonk use this docker image to run my ECS Task.

Also, I have a PG database hosted at AWS RDS. 

I am giving configs as an env variable on my task definition at ECS.
like:
![image](https://user-images.githubusercontent.com/17556123/210384050-3040268c-42e4-46dd-9196-44c1d9a100c5.png)

Is it possible to run (listmonk/listmonk:latest) container with only env variables? (Without config.toml file)

**Comment by knadh** [comment]
Yes. Eg: `LISTMONK_db__host=localhost LISTMONK_db__port=5432 ...      ./listmonk --config=''`

The env variables can be found in the docs: https://listmonk.app/docs/configuration/#environment-variables

**Comment by mericozkayagan** [comment]
Greetings, I have tried with the env variables but i am having an error. Logs are down below

`

2023-01-04T21:45:32.062+03:00 | 2023/01/04 18:45:32 main.go:97: v2.3.0 (448f0e3 2022-11-27T17:52:51Z)
-- | --
  | 2023-01-04T21:45:32.062+03:00 | 2023/01/04 18:45:32 init.go:132: reading config: config.toml
  | 2023-01-04T21:45:32.063+03:00 | 2023/01/04 18:45:32 init.go:259: connecting to db: listmonk.cs6c00irskxy.us-east-2.rds.amazonaws.com:5432/postgres
  | 2023-01-04T21:45:32.266+03:00 | 2023/01/04 18:45:32 main.go:137: the database does not appear to be setup. Run --install.

`

I have also tried --install and ./listmonk --install but it is not found
